src/utils/getComputedOcpOnAwsReportItems.ts

```diff
@@ -0,0 +1,138 @@
+import { OcpOnAwsQuery } from 'api/ocpOnAwsQuery';
+import {
+  OcpOnAwsDatum,
+  OcpOnAwsReport,
+  OcpOnAwsReportData,
+  OcpOnAwsReportValue,
+} from 'api/ocpOnAwsReports';
+import { Omit } from 'react-redux';
+import { sort, SortDirection } from './sort';
+
+export interface ComputedOcpOnAwsReportItem {
+  app?: string;
```

Comment:
What is `app` for? Is is just from when we had the tagging confusion?

Reply:
Most likely. I've removed it